src/SelfService/Infrastructure/Api/JsonSchema/SelfServiceJsonSchemaController.cs

```diff
@@ -22,10 +22,7 @@ public SelfServiceJsonSchemaController(ISelfServiceJsonSchemaService selfService
     [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status401Unauthorized, "application/problem+json")]
     [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status404NotFound, "application/problem+json")]
     [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status500InternalServerError, "application/problem+json")]
-    public async Task<IActionResult> GetSchema(
-        string id,
-        [FromQuery] int schemaVersion = ISelfServiceJsonSchemaService.LatestVersionNumber
-    )
+    public async Task<IActionResult> GetSchema(string id, [FromQuery] int schemaVersion)
```

Comment:
If we did something like
`[FromQuery]int? schemaVersion`

Would that allow us to check for `null` rather than for `0`?
Not sure if it is prettier

Reply:
not sure 🤷